server/guiserver/apps.py

```diff
@@ -108,7 +108,7 @@ def server():
         'jujugui.gzip': options.gzip,
         'jujugui.insecure': options.insecure,
         'jujugui.interactive_login': options.interactivelogin,
-        'jujugui.jem_url': options.jemurl,
+        'jujugui.bundleservice_url': options.bundleservice_url,
```

Comment:
Sorry that was from the options.py which parses the ini file and then views.py which generates the config.js 

This is from the views.py
```
'bundleServiceURL': settings['jujugui.bundleservice_url'],
```